docs/CSS_POLICY.md

```diff
@@ -0,0 +1,53 @@
+# CSS Policy
+
+We use the [GOVUK.UK Design System][govuk-ds] to provide as much as our styling as possible:
+
+- To benefit from their accessibility, performance and usability concerns
+- To be in-line with other GOV.UK services
+- To reduce our maintenance requirements
+- To be compliant with GDS standards
+
+Where possible, **GOV.UK Design System components and patterns should be used**.
+
+## Custom app CSS
+
+Where the GOV.UK Design System does not meet our needs, we _may_ write custom CSS for the DigiDeps app. When doing so, the following rules should be followed:
+
+- Reusable components should be designed as Twig templates or macros, so they can easily be updated in the future
```

Comment:
Worth adding something about how you break up the scss files (presumably by component, but it's not explicit) and any common naming conventions between scss and twig template.

Reply:
Yes, good shout. I'll also make sure our _existing_ files align with that.

Reply:
I've noted that in the policy, but we've got quite a few existing exceptions. I'll include that work in future tickets that continue this upgrade.